setup.py

```diff
@@ -105,6 +105,16 @@
             print("Error executing check_cblas.py - cblas not found")
     elif 'bdist_wheel' not in sys.argv:
         blas_info = get_info("blas_opt", 0)
+    elif platform.system() != 'Linux': # is mac
```

Comment:
Why don't we check for mkl on linux ?

Reply:
this issue is mostly about conda on mac, which comes with MKL, but we fail to find the headers